Admin/PageAdmin.php

```diff
@@ -45,6 +51,7 @@ protected function configureListFields(ListMapper $listMapper)
     protected function configureFormFields(FormMapper $formMapper)
     {
         parent::configureFormFields($formMapper);
+        $ivoryCKeditorAvailable = class_exists('Ivory\CKEditorBundle\IvoryCKEditorBundle') && $this->ivoryCkeditor;
```

Comment:
why not do this in the DI extension? ie. if a config is set, check if the Bundle is available .. if not throw an error.